tests/func/test_repo.py

```diff
@@ -50,16 +52,16 @@ class TestIgnore(TestDvcGit):
     def _stage_name(self, file):
         return file + Stage.STAGE_FILE_SUFFIX
 
-    def test_should_not_gather_stage_files_from_ignored_d(self):
+    def test_should_not_gather_stage_files_from_ignored_dir(self):
```

Comment:
@Suor this test seems to be artifical, and similar case as 
https://github.com/iterative/dvc/pull/2161#discussion_r301417177
I think i should remove that, what do you think?

Reply:
We are testing real behavior with code backing out here. If we break that this will show it